src/config.cpp

```diff
@@ -190,6 +193,50 @@ void parse_program_options(int argc, const char** argv)
         std::exit(EXIT_FAILURE); // hmm...
     }
 
+    // time synchronization
+    config.use_clockid = false;
+    if (!clock_name.empty())
+    {
+#if defined(USE_PERF_CLOCKID) && !defined(HW_BREAKPOINT_COMPAT)
+        struct clock_descripton
+        {
+            const char name[16];
```

Comment:
Is there a reason, why this isn't a std::string?

Reply:
I wanted the struct to be usable as `constexpr`, that won't work with `std::string`s. But even then I'd think `const char*` might be better than `const char[16]` :thinking:

But if you think using `std::string` is more idiomatic I'll change it.

Reply:
While `constexpr` certainly sounds like a valid point, it smells like premature optimization for me. 
`clock_description` is static anyways, so I don't see the benefit for `constexpr` here.
Using std::string is just less demanding in terms of brain computing capacity.